README.md

```diff
@@ -4,103 +4,99 @@ This package provides a set of language helpers, utility functions, and classes
 It includes APIs for feature detection, asynchronous and streaming operations, basic event handling,
 and making HTTP requests.
 
-## Features
+## Installation
 
-### Feature Detection
+This package is currently in Alpha with a initial stable release scheduled for later this year. You can install the Alpha
+through npm:
 
-Using the latest Web technologies isn't always as straightforward as developers would like due to differing support
-across platforms. `dojo-core/has` provides a simple feature detection API that makes it easy to detect which platforms
-support which features.
+```
+npm install dojo@2.0.0-alpha4
+```
+
+You can also clone or download this repository.
 
-#### Detecting Features
+## Dojo Usage
 
-The default export of `dojo-core/has` is a function which accepts a single parameter: the name of the feature to test for.
-If the feature is available, a truthy value is returned, otherwise a falsy value is returned:
+### npm
+
+If using npm, you can access modules by using ```require``` like so:
 
 ```ts
-if (has('dom-addeventlistener')) {
-    element.addEventListener('click', function () { /* ... */ });
-}
+import lang = require('dojo/lang');
 ```
 
-#### Adding Feature Detections
-
-It's important to be able to add new feature tests that aren't provided out-of-the-box by `dojo-core/has`.
-This can be done easily by using the `add` function exported by the `has` module. It accepts two parameters:
-the name of the feature, and either an immediate value indicating its availability or a function that resolves to a
-value.
+### Downloading
```

Comment:
The heading "Downloading" doesn't really seem to fit this section, which is about module loading. It should also be mentioned that this syntax is specific to ES6 and TS.